tests/brevitas/nn/test_recurrent.py

```diff
@@ -194,3 +282,46 @@ def test_quant_lstm_fwd_call(
             cat_output_cell_states=shared_cell_state_quant,
             return_quant_tensor=return_quant_tensor)
         assert m(inp) is not None
+
+    @pytest.mark.parametrize("batch_first", [True, False])
+    @pytest.mark.parametrize("bidirectional", [True, False, 'shared_input_hidden_weights'])
+    @pytest.mark.parametrize("num_layers", [1, 2])
+    @pytest.mark.parametrize("bias", [True, False])
+    @pytest.mark.parametrize("coupled_input_forget_gates", [True, False])
+    @pytest.mark.parametrize("return_quant_tensor", [True, False])
+    @pytest.mark.parametrize("shared_intra_layer_weight_quant", [True, False])
+    @pytest.mark.parametrize("shared_intra_layer_gate_acc_quant", [True, False])
+    @pytest.mark.parametrize("shared_cell_state_quant", [True, False])
+    @pytest.mark.skip("FIXME: many warnings due to __torch_function__")
```

Comment:
note I'm disabling this test for now since there are many __torch__function warnings that need fixing